DataFormats/Math/src/classes.h

```diff
@@ -225,9 +224,6 @@ namespace DataFormats_Math {
     ROOT::Math::MatRepStd<double, 10 , 10> smdcw;
     ROOT::Math::MatRepStd<double, 2 , 3> smdcw1;
 
-    //Used by MET Significance matrix
-    ROOT::Math::SMatrix<double,2> smat;
```

Comment:
OK, I recall this https://github.com/cms-sw/cmssw/pull/7955#issuecomment-76074944
As long as  74X is ROOT5, this change will break the feature discussed in https://github.com/cms-sw/cmssw/pull/7866